tests/services/test_file_mover.py

```
@@ -288,3 +288,47 @@ def custom_exists_side_effect(path_obj):
     mock_path_unlink.assert_called_once_with(dest)
+
+def test_move_symlink_direct(tmp_path):
+    """Test _move_symlink when it points to an absolute path."""
+    target = tmp_path / "actual_target.txt"
+    target.write_text("data")
+    link = tmp_path / "the_link"
+    link.symlink_to(target)
+    dest = tmp_path / "final_dest.txt"
+    
+    from app.services.file_mover import _move_symlink
+    success, error = _move_symlink(link, dest)
```

Ruff (0.15.2)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)